modules/highgui/src/window_cocoa.mm

```diff
@@ -944,8 +944,6 @@ - (void)createSliderWithName:(const char *)name maxValue:(int)max value:(int *)v
     [[slider name] setStringValue:cvname];
     [[slider slider] setMaxValue:max];
     [[slider slider] setMinValue:0];
```

Comment:
Should this be `min`?

Reply:
I guess this should be a different issue.

Reply:
In general yes, it should be `min`.
In "createSliderWithName", we always have `min = 0`

Reply:
Ah, I see that now.